django/db/backends/sqlite3/features.py

```diff
@@ -30,13 +30,9 @@ class DatabaseFeatures(BaseDatabaseFeatures):
     supports_temporal_subtraction = True
     ignores_table_name_case = True
     supports_cast_with_precision = False
-    uses_savepoints = Database.sqlite_version_info >= (3, 6, 8)
-    supports_index_column_ordering = Database.sqlite_version_info >= (3, 3, 0)
+    uses_savepoints = True
     can_release_savepoints = uses_savepoints
-    can_share_in_memory_db = (
-        Database.__name__ == 'sqlite3.dbapi2' and
-        Database.sqlite_version_info >= (3, 7, 13)
-    )
+    can_share_in_memory_db = Database.__name__ == 'sqlite3.dbapi2'
```

Comment:
The `Database.__name__ == 'sqlite3.dbapi2'` condition is from 8c99b7920e8187f98cf4d7dbd9918bd6c6da1238. I'm not sure if it's still needed (when would it return `False`?) \cc @coldmind